spec/eventception/dispatcher_spec.rb

```diff
@@ -2,10 +2,14 @@
 require 'support/listener'
 require 'support/subscriber'
 require 'support/test_event'
+require 'eventception/event_handler'
 
 describe Eventception::Dispatcher do
+  let(:dispatcher) { described_class.new }
```

Comment:
It's basically the same, but you can make it a little more obvious by doing `subject(:dispatcher) { described_class.new }` (and you can still use the `dispatcher` name below)